actions/gallery_actions.js

```diff
@@ -152,63 +93,17 @@ export const getPhotosFromCameraroll = () => async (dispatch, getState) => {
                     width: item.node.image.width,
                     lat: item.node.location.latitude,
                     lon: item.node.location.longitude,
-                    timestamp: item.node.timestamp,
+                    date: item.node.timestamp,
                     selected: false,
-                    picked_up: false,
```

Comment:
Why are we not initialising this?

Reply:
Because this is when we load geotagged cameraroll/gallery images to state, we don't need to add picked_up here..
We add pickedUp in `ADD_IMAGES` i.e when we select gallery image